docs/_static/theme_overrides.css

```diff
@@ -1,7 +1,38 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
 
  .wy-table-responsive table td {
     /* !important prevents the common CSS stylesheets from overriding
        this as on RTD they are loaded after this stylesheet */
     white-space: normal !important;
     vertical-align: top !important;
  }
+
+.material-icons {
+    display: inline-flex;
+    vertical-align: middle !important;
+    padding-bottom: 8px !important;
+    padding-right: -5px !important;
+}
+
+.local_library {
+	color: #2196f3;
+}
+
+.schedule {
+	padding-bottom: 1px !important;
+	background-color: #2196f3;
+	border-radius: 50%;
+	color: #fff;
+}
+
+.star {
+	padding-bottom: 1px !important;
+	background-color: #ffc107;
+	border-radius: 50%;
+	color: #fff;
+}
+
+img {
+	height: 1.2em;
```

Comment:
This affects all images, but is it what we want? It will make a smaller image stretched beyond its resolution.

There's already `max-width: 100%`, so I would suggest deleting this.

Reply:
What was the purpose of this?

Reply:
It affects only inline images and was introduced for better spacing.

Without:
![2018-07-06_6-49-51](https://user-images.githubusercontent.com/1457929/42360704-c55719ec-80ea-11e8-9ec3-0a012ad61764.png)

With:
![2018-07-06_6-45-50](https://user-images.githubusercontent.com/1457929/42360714-d4fbc7a8-80ea-11e8-9782-0ed1499840b2.png)